Minor editorial, grammatical, and spelling improvments to docs (#1701)
vercel/turborepo: docs/pages/docs/guides/monorepo-tools.mdx

Below is a running list of additional monorepo tools that you may find useful in your turborepo.

💯 = stuff we use or have used in [Turborepo's own monorepo](https://github.com/vercel/turborepo).

## Versioning, Publishing, and Changelog Generation

> For the foreseeable future, `turbo` is not going manage package publishing. Versioning/Publishing is an extremely opinionated topic with a lot of existing solutions. Our advice is to avoid versioning packages altogether unless you actually need to publish them to npm for external consumption. If you have to version your packages, we really like the workflow of Changesets (especially for open source projects).

Reviewer: ```suggestion
For the foreseeable future, `turbo` will not manage package publishing. Versioning/Publishing is an extremely opinionated topic with many existing solutions. We advise avoiding versioning packages altogether unless you need to publish them to npm for external consumption. If you have to version your packages, we like the workflow of Changesets (especially for open source projects).
```
